meson.build

```diff
@@ -212,7 +212,10 @@ else
   gudev = dependency('', required : false)
 endif
 libxmlb = dependency('xmlb', version : '>= 0.1.13', fallback : ['libxmlb', 'libxmlb_dep'])
-gusb = dependency('gusb', version : '>= 0.3.5', fallback : ['gusb', 'gusb_dep'])
+if get_option('gusb')
+  gusb = dependency('gusb', version : '>= 0.3.5', fallback : ['gusb', 'gusb_dep'])
+  conf.set('HAVE_GUSB', '1')
+endif
```

Comment:
I think win32 build really doesn't make sense unless you have gusb - that's the only plugins we can do.
t probably should be set to fail out without usb support.

Reply:
Fixed, good idea.